lando/server/jobapi.py

```diff
@@ -181,14 +180,13 @@ def get_input_files(self):
 
     def get_credentials(self):
         """
-        Get all dds user/app credentials attached to this job.
-        :return: Credentials: let's user lookup credential info based on bespin user/app ids.
+        Get all bespin service account credentials.
+        :return: Credentials: bespin DukeDS credentials
         """
         job = self.get_job()
-        user_id = job.user_id
         credentials = Credentials()
 
-        for user_credential_data in self.api.get_dds_user_credentials(user_id):
+        for user_credential_data in self.api.get_dds_user_credentials():
```

Comment:
The key to each credential is stored with each file. The worker has to lookup the credential from a dictionary based on the credential key in each file. In theory there could be multiple DukeDS stores so we could end up reading files from both and needing to send multiple credentials.